test/api_container_create_test.go

```diff
@@ -302,3 +302,23 @@ func (suite *APIContainerCreateSuite) TestAliOSOptions(c *check.C) {
 	c.Assert(err, check.IsNil)
 	CheckRespStatus(c, resp, 201)
 }
+
+func (suite *APIContainerCreateSuite) TestCreateOOMOption(c *check.C) {
+	cname := "TestCreateOOMOption"
+	q := url.Values{}
+	q.Add("name", cname)
+	query := request.WithQuery(q)
+
+	obj := map[string]interface{}{
+		"Image": busyboxImage,
+		"HostConfig": map[string]interface{}{
+			"OomScoreAdj":    100,
+			"OomKillDisable": true,
+		},
+	}
+	body := request.WithJSONBody(obj)
+
+	resp, err := request.Post("/containers/create", query, body)
+	c.Assert(err, check.IsNil)
+	CheckRespStatus(c, resp, 201)
```

Comment:
It tested that the container creation works. Please test further to check if the ContainerMeta contains this field with the inspect API.

Reply:
this will be test in `test/cli_create_test.go`